.ci/values.yaml

```diff
@@ -65,6 +70,11 @@ check:
       tag: v3.2.1
     extraEnvs:
     nodeSelector: {}
+    tolerations:
```

Comment:
maybe put `tolerations: []` so we know it takes in a list?

Reply:
Updated, now with brackets! 😃